### ISSUE TITLE
CNN hyperparameter tuning

### ISSUE THREAD

**Issue by shok0n** [shok0n/remote-sensing-lc#10]
Choose one CNN architecture or training hyperparameter (e.g. learning rate, batch size, number of filters, dropout, epochs). Justify the choice and value, state an expectation, implement, and compare results to the expectation.
